code/modules/antagonists/revolution/revolution.dm

```diff
@@ -368,7 +369,7 @@
 			var/list/datum/mind/nonhuman_promotable = list()
 			for(var/datum/mind/khrushchev in non_heads)
 				if(khrushchev.current && !khrushchev.current.incapacitated() && !HAS_TRAIT(khrushchev.current, TRAIT_RESTRAINED) && khrushchev.current.client)
-					if(ROLE_REV in khrushchev.current.client.prefs.be_special)
+					if(ROLE_REV_HEAD in khrushchev.current.client.prefs.be_special)
```

Comment:
Check provocateur in here too

Reply:
Shouldn’t migrate_antagonist already take care of this?

Reply:
`migrate_antagonist` *created* the provocateur preference.

The point is that people with rev head *or* provocateur should be able to be promoted.

Reply:
I see I see, did it now